src/views/studio/MyVideosView/MyVideosView.tsx

```diff
@@ -254,7 +254,7 @@ export const MyVideosView = () => {
         }
         return (
           <VideoTilePublisher
-            key={idx}
+            key={`video-tile-${video.id || idx}`}
```

Comment:
I think this may have some undesired consequences when video IDs are matching some of the indexes. Check this out:

https://user-images.githubusercontent.com/12646744/135095096-ff46b46a-dc0a-4259-838f-39836269b9fe.mov

Can you see the `banana` video flashing for a second when switching the tabs? It's because it has an ID of `1` so React tries to reuse that same tile when loading the tile with index `1`. I think this will probably help but you need to test it out:
```suggestion
            key={video.id ? `video-id-${video.id}` : `video-idx-${idx}`}
```

Reply:
@kdembler Yeah, you are right, good catch 👍. By the way, we have more places in app where we use `index` as key. What do you think we fix all these places and add eslint rule for that?

https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-array-index-key.md

Reply:
Not sure, I think the rule may be a bit too generic, I think there are still some places in the app where it makes sense to use index as the key, I think we just shouldn't mix those with IDs